_rules/role-has-required-states-and-properties-4e8ab6.md

```diff
@@ -18,7 +18,7 @@ authors:
 
 ## Applicability
 
-Any HTML or SVG element that has an [explicit semantic role](#semantic-role), except if the element has an [implicit semantic role](#implicit-role) that is identical to the explicit semantic role.
+Any HTML or SVG element that has an [explicit semantic role](#semantic-role), except if the element has an [implicit semantic role](#semantic-role) that is identical to the explicit semantic role.
```

Comment:
This definitely isn't the way to solve it. I think we need a definition for this.

Reply:
Then, do we also need a definition for "explicit semantic role" which is used in the very same rule and simply links to "semantic role"?

The definition of semantic role has mention of both implicit and explicit roles, with links to further explanations. I assumed that if it was OK to link "explicit semantic role" to it, then it would also be OK for "implicit semantic role".

Reply:
@WilcoFiers I do not understand why it is fine to have `[explicit semantic role](#semantic-role)` (as was here before), but it is not fine to have `[implicit semantic role](#semantic-role)` Especially given that the glossary entry for semantic role mention both implicit and explicit roles, with links to more details.

Could you explain why one is OK and the other is not?
Should we rather add definitions for both explicit and implicit roles? (they are going to essentially be the same links that already exist in the definition of "Semantic Role")

Reply:
You're right. We shouldn't have linked explicit semantic role to semantic role either. I'll open an issue for it, as to not block this PR any further.